packages/evo-marko/src/tags/evo-ccd/index.marko

```diff
@@ -0,0 +1,58 @@
+import type { AttrString } from "marko/tags-html";
+export interface Input extends Marko.HTML.Div {
+    max?: string;
+    min?: string;
+    chargerIcon?: "included" | "not-included";
+    units?: AttrString;
+    /**
+     * The aria-label accessibility label for the ccd component.
+     * This is for internationalization.
+     * It should use min, max, and charger included or not included, and secondaryText in the label in order to demonstrate
+     * to screen readers the content on the component.
+     * Expected return `Charger included. ${min} - ${max} Watts. USB PD`
+     */
```

Comment:
IntelliSense displays this as Markdown, so if your intention is to have multiple paragraphs instead of one big block of text they need to be separated by a blank line